python/applications/ospuller/client.py

```diff
@@ -0,0 +1,55 @@
+﻿#!/usr/bin/python
+'''
+Created on Dec 17, 2015
+
+@author: Arthur Valadares
+'''
+
+import logging
+import logging.handlers
+import os
+import sys
+from applications.ospuller.ospuller import OpenSimPuller
+from applications.ospuller.testsim import OpenSimPullerTestSimulation
+
+sys.path.append(os.path.realpath(os.path.join(os.path.dirname(__file__), "../..")))
+
+from applications.nodesim.nodesim import NodeSimulation
+from applications.nodesim.testsim import NodeTestSimulation
+from spacetime_local.frame import frame
+
+class Simulation(object):
+    '''
+    classdocs
+    '''
+
+    def __init__(self):
+        '''
+        Constructor
+        '''
+        frameos = frame(time_step=200)
+        frameos.attach_app(OpenSimPuller(frameos))
+
+        frametest = frame(time_step=200)
+        frametest.attach_app(OpenSimPullerTestSimulation(frametest))
+
+        frameos.run_async()
+        frametest.run_async()
```

Comment:
It's better to have something working out of the box.. they can comment it out if they wish to try it with another simulation.